### PR TITLE
Consolidate module_loc and module_org fixtures

### PR DESCRIPTION
**Pull request by mshriver**
rename to module_location

create new shared smart_proxy_location module

remove all the duplicate fixture defintions, mostly in the UI test
modules

### CODE REVIEW

**Review by JacobCallahan — APPROVED** [review]

File: tests/foreman/ui/test_discoveredhost.py [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 
 @pytest.fixture(scope='module')
-def module_org():
+def discovery_org():
     org = entities.Organization().create()
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_discoveredhost.py:33]:
```suggestion
    org = default_sat.api.Organization().create()
```

File: tests/foreman/ui/test_discoveredhost.py [hunk 2]
```diff
@@ -42,8 +42,8 @@ def module_org():
 
 
 @pytest.fixture(scope='module')
-def module_loc(module_org):
-    loc = entities.Location(name=gen_string('alpha'), organization=[module_org]).create()
+def discovery_location(discovery_org):
+    loc = entities.Location(name=gen_string('alpha'), organization=[discovery_org]).create()
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_discoveredhost.py:46]:
```suggestion
    loc = default_sat.api.Location(name=gen_string('alpha'), organization=[discovery_org]).create()
```

File: pytest_fixtures/api_fixtures.py [hunk 3]
```diff
@@ -88,6 +88,17 @@ def module_location(module_org):
     return entities.Location(organization=[module_org]).create()
 
 
+@pytest.fixture(scope='module')
+def smart_proxy_location(module_org, default_sat):
+    location = entities.Location(organization=[module_org]).create()
```
Comment (Gauravtalreja1) [inline pytest_fixtures/api_fixtures.py:93]:
```suggestion
    location = default_sat.api.Location(organization=[module_org]).create()
```

File: tests/foreman/ui/test_discoveredhost.py (hunk 1 above)
Comment (mshriver) [reply tests/foreman/ui/test_discoveredhost.py:33]: I changed some of these as I went, where there were only a few references in the module and I could change all of them and remove the import. If its alright with you, can I hold these additional changes to make in a bulk update against `master` branch?

Comment (Gauravtalreja1) [reply tests/foreman/ui/test_discoveredhost.py:33]: Yes, changing this in master for all and removing entities import, sounds good. 👍

**Review by Gauravtalreja1 — APPROVED** [review]

**Comment by mshriver** [comment]
merging with 2 approvals

**Pull request merged**